stories/module-analytics.stories.js

```diff
@@ -124,6 +127,44 @@ storiesOf( 'Analytics Module', module )
 			</WithTestRegistry>
 		);
 	} )
+	.add( 'Property Select including GA4 properties', () => {
+		const { accounts, properties, profiles } = fixtures.accountsPropertiesProfiles;
+		// eslint-disable-next-line sitekit/acronym-case
+		const accountID = properties[ 0 ].accountId;
+		// eslint-disable-next-line sitekit/acronym-case
+		const propertyID = profiles[ 0 ].webPropertyId;
+		const setupRegistry = ( { dispatch } ) => {
+			dispatch( STORE_NAME ).receiveGetAccounts( accounts );
+			dispatch( STORE_NAME ).finishResolution( 'getAccounts', [] );
+
+			// eslint-disable-next-line sitekit/acronym-case
+			dispatch( STORE_NAME ).receiveGetProperties( properties, { accountID: properties[ 0 ].accountId } );
+			dispatch( STORE_NAME ).receiveGetProfiles( profiles, {
+				accountID,
+				// eslint-disable-next-line sitekit/acronym-case
+				propertyID,
+			} );
+
+			dispatch( STORE_NAME ).receiveGetSettings( {
+				accountID,
+			} );
+			// eslint-disable-next-line sitekit/acronym-case
+			dispatch( MODULES_ANALYTICS_4 ).receiveGetProperties(
+				propertiesGA4,
+				{ accountID }
+			);
+		};
+
+		return (
+			<WithTestRegistry callback={ setupRegistry }>
+				<SetupWrap>
+					<div className="googlesitekit-setup-module__inputs">
+						<PropertySelectIncludingGA4 />
+					</div>
+				</SetupWrap>
+			</WithTestRegistry>
+		);
```

Comment:
Tested and working https://google.github.io/site-kit-wp/storybook/pull/3383/?path=/story/analytics-module--property-select-including-ga4-properties